_pages/vot-trust-framework.md

```diff
@@ -0,0 +1,62 @@
+---
```

Comment:
This filename here is intended to match up to the trustmark we have published in the repo: https://github.com/18F/identity-idp/blob/5837343cd6c1a9fe3f78d56d9931808e108e80a0/config/application.yml.default#L344